### DISCUSSION TITLE
Font family rename for subsets

### DISCUSSION THREAD

**Discussion by hansagames** [fonttools/fonttools#2574] [Q&A] [answered]
Hi,

was trying to create subset from font and could not find a way to rename font name for that subset. As it have OFL license and Reserved Font Name, its necessary to rename font if you make subset out of it (as fare as discussions on RFN says).
Is that supported feature or I need to manually rename font using script like this https://github.com/fonttools/fonttools/blob/main/Snippets/rename-fonts.py or some other tool ? 

Thank you

**Comment by anthrotype** [comment, marked as answer]
yeah, currently there's no option for that, you'd have to rename it afterwards. Though it could be a nice feature to have. Maybe incorporating some of the logic from that rename-fonts snippet. Also other tools (the pyftmerge, varLib.instancer, etc.) could take advantage of it. PR welcome

**Reply by anthrotype** [reply]
on second thought, maybe it's preferable to have a distinct tool (e.g. `fonttools rename`) that does that istead of overloading the subsetter with extra options that don't have to do strictly with subsetting operation, following "do one thing, do it well" philosophy
